src/tests/frontend/cypress/cypress/README.md

```diff
@@ -0,0 +1,10 @@
+# Cypress Etherpad guide
```

Comment:
Why the `cypress/cypress` directory structure? Why not move everything in `src/tests/frontend/cypress/cypress` to `src/tests/frontend/cypress`?

Reply:
```
C:\etherpad-lite\src\tests\frontend\cypress>cypress run ..\integration\test.js --config-file cypress.json

Can't run because no spec files were found.

We searched for any files inside of this folder:

C:\etherpad-lite\src\tests\frontend\cypress\cypress\integration
```

I will run the binary from a level up but it's messy ;\